connector/dao/src/main/java/it/pagopa/pdv/tokenizer/connector/dao/TokenizerConnectorImpl.java

```diff
@@ -77,6 +79,9 @@ private String saveGlobalToken(String pii) {
                         GlobalFiscalCodeToken.Fields.status);
             }
         }
+        catch(ProvisionedThroughputExceededException e){
+            throw new TooManyRequestsException(e.getCause());
```

Comment:
you have to set the "current" exception as the cause of the new exception (do this to all refs)
```suggestion
            throw new TooManyRequestsException(e);
```